sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/XmlParserHelper.java

```diff
@@ -38,19 +37,19 @@
   private final InputStreamReader reader;
   private final XMLStreamReader stream;
 
-  public XmlParserHelper(File file) {
+  XmlParserHelper(File file) {
     try {
       this.file = file;
       this.reader = new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8);
       XMLInputFactory xmlFactory = XMLInputFactory.newInstance();
       this.stream = xmlFactory.createXMLStreamReader(reader);
 
     } catch (FileNotFoundException | XMLStreamException e) {
-      throw Throwables.propagate(e);
+      throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)